fluent-dom/makefile

```diff
@@ -5,31 +5,27 @@ DEPS    := cached-iterable:CachedIterable
 include ../common.mk
 
 lint:
-	@eslint --config $(ROOT)/eslint_js.json --max-warnings 0 src/
-	@eslint --config $(ROOT)/eslint_test.json --max-warnings 0 test/
+	@$(ESLINT) --config $(ROOT)/eslint_js.json --max-warnings 0 src/
+	@$(ESLINT) --config $(ROOT)/eslint_test.json --max-warnings 0 test/
 	@echo -e " $(OK) lint"
 
 .PHONY: compile
 compile: esm/.compiled
 
 esm/.compiled: $(SOURCES)
-	@tsc
+	@$(TSC)
 	@touch $@
 	@echo -e " $(OK) esm/ compiled"
 
 .PHONY: test
 test: esm/.compiled
-	@nyc --reporter=text --reporter=html mocha \
-	    --recursive --ui tdd \
-	    --require esm \
-	    --require ./test/index \
-	    test/**/*_test.js
+	@$(MAKE) mocha-test TEST_REQUIRES="--require ./test/index"
```

Comment:
Perhaps drop `TEST_REQUIRES` and leave the full command here as a special case, same as you did in `fluent-react`?

Reply:
I left this in. This is a pretty common practice to customize build recipes in the world of make.